packages/wxt/src/core/utils/globals.ts

```diff
@@ -12,7 +12,12 @@ export function getGlobals(
     {
       name: 'BROWSER',
       value: config.browser,
-      type: `string`,
+      type:
+        config.targetBrowsers.length === 0
+          ? 'string'
+          : config.targetBrowsers
+              .map((browser) => JSON.stringify(browser))
```

Comment:
Let's be a little bit more careful here. If someone where to enter an object instead of a string, I would prefer to show `"[Object object]"` over `{ ... }` as the type. Basically, we need to ensure the type will always be a string.

```suggestion
              .map((browser) => `"${browser}"`)
```